src/components/Form/SignInModal/index.tsx

```diff
@@ -0,0 +1,25 @@
+import SignUp from '@components/Form/SignInModal/SignUp'
+import SignIn from '@components/Form/SignInModal/SignIn'
+import { useState } from 'react'
+import { Modal } from '@components/Modal'
```

Comment:
Simplificar imports

Reply:
I make a better way of import modules in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/35/commits/c51d00664c6dae3875c8b8052f0a073cebda6135)